tests/lcs/core/interface/snapshot.livecodescript

```diff
@@ -0,0 +1,53 @@
+script "CoreInterfaceSnapshot"
+/*
+Copyright (C) 2015 LiveCode Ltd.
+
+This file is part of LiveCode.
+
+LiveCode is free software; you can redistribute it and/or modify it under
+the terms of the GNU General Public License v3 as published by the Free
+Software Foundation.
+
+LiveCode is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or
+FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+for more details.
+
+You should have received a copy of the GNU General Public License
+along with LiveCode.  If not see <http://www.gnu.org/licenses/>.  */
+
+on TestSetup
+  if the environment is "command line" then
+      return "SKIP GUI-only tests"
+  end if
+end TestSetup
+
+on TestImportSnapshotFromRect
+    import snapshot from rectangle 100,100,500,400
+    
+    TestAssert "test set it", it is the long id of the last image
+end TestImportSnapshotFromRect
+
+on TestImportSnapshotFromObjectRect
+    create group "testGroup"
+
+    import snapshot from rectangle (the rect of group "testGroup")
+    TestAssert "test set it", it is the long id of the last image
+end TestImportSnapshotFromObjectRect
+
+
+on TestImportSnapshotFromObject
```

Comment:
This and the next both should not need GUI mode - as they rasterize directly, and the screen (GUI) is not required at all.

(The difference is in -ui mode, MCscreen is an instance of MCUIDC, which isplatform agnostic - and not MCScreenDC, which is per-platform and contains things which interact with the GUI subsytem of the platform the engine is running on).